File: csharp/LifetimeIntegration/AuthenticationExample.cs
using System;
using LifetimeIntegration.LifetimeAuthentication;

namespace LifetimeIntegration {
    class AuthenticationExample {
        
        // Get a session token to avoid sending the credentials on every request
        public static String getToken() {
                        
            // LifeTime credentials used to obtain a session token
            const string username = "user";
            const string password = "password";

            // Authentication Service WS proxy created by Visual Studio
            AuthenticationService service = new AuthenticationService();

            // The status of the WS call
            bool success = false;
            APIStatus status = null;
            

            // Authentication token to be used when invoking other services
            // WS returns a boolean and status code to signal success
            string token = service.Authentication_GetToken(username, password, out status, out success);

            // If the call was successful, print the session token obtained
            if (success) {
                Console.Write(String.Format("Token obtained: {0}", token));
            } else {
                // Implement error handling by checking the status.ResponseId field
                // See the possible error codes in the APIStatus structure documentation
            }
            return token;
        }
    }
}

// This example displays the output:
// Token obtained 96fef1f3-f20c-4254-98f7-02d6fb50cdf7

File: java/LifetimeIntegration/src/AuthenticationExample.java
import javax.xml.ws.Holder;
import lifetime.authentication.*;

public class AuthenticationExample {

	// Obtain a session token to avoid sending the credentials on every request
	public static void main(String[] args) {
		GetToken();
	}
	
	public static String GetToken() {
 
		// LifeTime credentials used to obtain a session token
        String username = "username";
        String password = "password";
        
        // Objects to store the values returned
        Holder<String> token = new Holder<String>();
		Holder<APIStatus> status = new Holder<APIStatus>();
		Holder<Boolean> success = new Holder<Boolean>(); 
        
		// Authentication Service WS proxy created by Oracle wsimport tool
        AuthenticationService service = new AuthenticationService();
        service.getAuthenticationServiceSoap().authenticationGetToken(username, password, success, status, token);
        
        if(success.value) {
        	// System.out.println(String.format("Token obtained %s", token.value));
        } else {
			// Implement error handling. 
			// See the possible error codes in the APIStatus structure documentation
		}
        return token.value;
	}
}
// This example displays the output:
// Token obtained 96fef1f3-f20c-4254-98f7-02d6fb50cdf7